FantasmoSDK/app/src/main/java/com/fantasmo/sdk/views/FMParkingView.kt

```diff
@@ -401,20 +401,10 @@ class FMParkingView @JvmOverloads constructor(
             }
 
             override fun anchored(frame: Frame): Boolean {
-                var anchored = false
                 frame.let {
-                    if (frame.camera.trackingState == TrackingState.TRACKING) {
-                        fmLocationManager.setAnchor(it)
-                        anchored = true
-                    } else {
-                        Toast.makeText(
-                            context.applicationContext,
-                            "Anchor can't be set because tracking state is not correct, please try again.",
-                            Toast.LENGTH_LONG
-                        ).show()
-                    }
+                    fmLocationManager.setAnchor(it)
+                    return true
```

Comment:
Is it possible that `fmLocationManager.setAnchor(it)` can somehow fail? I see it has a void return type, and that the test for this method asserts that the anchor frame is not null. Is it safe to just return true or should we be doing some check here instead?